protocol/lavasession/consumer_session_manager_test.go

```diff
@@ -28,45 +30,75 @@ const (
 	firstEpochHeight                   = 20
 	secondEpochHeight                  = 40
 	cuForFirstRequest                  = uint64(10)
-	grpcListener                       = "localhost:28353"
 	servicedBlockNumber                = int64(30)
 	relayNumberAfterFirstCall          = uint64(1)
 	relayNumberAfterFirstFail          = uint64(1)
 	latestRelayCuAfterDone             = uint64(0)
 	cuSumOnFailure                     = uint64(0)
 )
 
+// This variable will hold grpc server address
+var grpcListener = "localhost:28353"
+
 func CreateConsumerSessionManager() *ConsumerSessionManager {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
removing the rand seed makes session id selection deterministic

Reply:
Returned c5b912d7298e0942b6ffffcbc62ab424ec181639